client/client-multi/client-c/src/kaa/kaa_notification_manager.c

```diff
@@ -588,6 +619,7 @@ kaa_error_t kaa_add_optional_notification_listener(kaa_notification_manager_t *s
         KAA_LOG_WARN(self->logger, err, "Failed to calculate optional listener id");
         return err;
     }
+    KAA_LOG_TRACE(self->logger, KAA_ERR_NONE, "Going to add optional notification listener id '%u', topic id '%u'", id, *topic_id);
```

Comment:
Please, remove this and rephrase following to 
KAA_LOG_TRACE(self->logger, KAA_ERR_NONE, "Added optional notification listener id '%u', topic id '%u'", id, *topic_id);
Thanks